### PR TITLE
Added task-list component

### PR DESCRIPTION
**Pull request by carmina-c**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -9,12 +9,14 @@ import { AppRoutingModule } from "./app-routing.module";
 import { AppComponent } from "./app.component";
```
Comment (Catalin12) [inline src/app/app.module.ts:9]:
```suggestion
import { AppComponent } from "./app.component";

```

File: src/app/task-list/task-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from "@angular/core";
```
Comment (Catalin12) [inline src/app/task-list/task-list.component.ts:1]:
```suggestion
import { Component } from "@angular/core";
```

File: src/app/task-list/task-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+				description: "Description 2",
+				isCompleted: false,
+				isDeleted: false,
+				userId: 1
+			}
+		];
+	}
+
+	public ngOnInit(): void {
+	}
```
Comment (Catalin12) [inline src/app/task-list/task-list.component.ts:36]:
```suggestion
```

File: src/app/task/task.component.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit } from "@angular/core";
```
Comment (Catalin12) [inline src/app/task/task.component.ts:1]: same here (OnInit)

File: src/app/task-list/task-list.component.ts (hunk 3 above)
Comment (carmina-c) [reply src/app/task-list/task-list.component.ts:36]: i know that at the moment there's nothing inside ngOnInit, but we'll have to load the tasks here once we have the api

File: src/app/task/task.component.ts [hunk 5]
```diff
@@ -9,34 +9,27 @@ import { TaskDTO } from "../shared/TaskDTO";
 })
 export class TaskComponent implements OnInit {
 
-	public taskTest: TaskDTO;
+	@Input()
+	public task: TaskDTO = {};
 	public areTaskDetailsShown: boolean = false;
```
Comment (danielursu) [inline src/app/task/task.component.ts]:
```suggestion
	@Input()
	public task: TaskDTO = {};
	
	public areTaskDetailsShown: boolean = false;
```

**Review by danielursu — APPROVED** [review]

File: src/app/app.module.ts (hunk 1 above)
Comment (carmina-c) [reply src/app/app.module.ts:9]: that's an internal import, so i think it's ok without the extra space

**Pull request merged**